### PR TITLE
fix issue #1979 emails

### PR DESCRIPTION
**Pull request by dev-101**
#1979 contact email account used for admin notifications

### CODE REVIEW

File: oc-includes/osclass/emails.php [hunk 1]
```diff
@@ -706,18 +732,24 @@ function fn_email_new_comment_admin($aItem) {
...
-            'from'      => osc_contact_email(),
-            'to'        => $admin_email,
-            'to_name'   => __('Admin mail system'),
-            'subject'   => $title_email,
-            'body'      => $body_email,
-            'alt_body'  => $body_email
-        );
-        osc_sendMail($emailParams);
+        $admins = Admin::newInstance()->listAll();
+        foreach($admins as $admin) {
```
Comment (conejoninja) [inline oc-includes/osclass/emails.php]: why loop the sendMail function instead of using an array in the "to" param (or bcc or cc)?

Comment (dev-101) [inline oc-includes/osclass/emails.php]: good catch ;)

Comment (dev-101) [inline oc-includes/osclass/emails.php]: I'll fix it later (cannot type right now), but if you wish to merge it now, you can do the fix

Comment (dev-101) [inline oc-includes/osclass/emails.php]: Hi Conejo, finally got at my PC and had time to look into this and expalin to you - why.
The reason is simple, in all other functions we send mails to users, not admins. But, in this case, we send them to admins only, and the "to" field would be empty (or the first admin from the instance would get 2 emails, unless I skip the array).

I know it seems like redundancy in the osc_send_mail() functions, but the code is very clear to read. If you think that it is better to use cc/bcc (can "to" field be empty in this case?), you can do the modification.

**Pull request merged**